modules/gsp-cluster/vpc-flow-logs.tf

```diff
@@ -0,0 +1,67 @@
+// TODO add variables somewhere
+
+resource "aws_flow_log" "vpc_flow_log" {
+  iam_role_arn         = "${var.cluster_name}_cloudwatch_vpc_flow_log_shipping_role"
+  log_destination      = "${aws_cloudwatch_log_group.vpc_flow_logs.arn}"
+  log_destination_type = "cloud-watch-logs"
+  traffic_type         = "ALL"
+  vpc_id               = "${var.vpc_id}"
+}
+
+
+resource "aws_cloudwatch_log_group" "vpc_flow_logs" {
+  name              = "${var.cluster_domain}_vpc_flow_log"
+  retention_in_days = 30
+}
+
+
+resource "aws_iam_role" "cloudwatch_vpc_flow_log_shipping_role" {
+  name = "${var.cluster_name}_cloudwatch_vpc_flow_log_shipping_role"
+
+  assume_role_policy = "${data.aws_iam_policy_document.cloudwatch_vpc_flow_log_assume.json}"
+}
+
+
+resource "aws_iam_policy" "cloudwatch_vpc_flow_log_shipping_policy" {
+  name        = "${var.cluster_name}_cloudwatch_vpc_flow_log_shipping_policy"
+  description = "Send logs to Clouwatch"
+
+  policy = "${data.aws_iam_policy_document.cloudwatch_vpc_flow_log.json}"
+}
+
+
+resource "aws_iam_policy_attachment" "cloudwatch_vpc_flow_log_shipping_policy" {
+  name       = "${var.cluster_name}_cloudwatch_vpc_flow_log_shipping_role_policy_attachement"
+  roles      = ["${aws_iam_role.cloudwatch_vpc_flow_log_shipping_role.name}"]
+  policy_arn = "${aws_iam_policy.cloudwatch_vpc_flow_log_shipping_policy.arn}"
+}
+
+
+data "aws_iam_policy_document" "cloudwatch_vpc_flow_log_assume" {
```

Comment:
`assume_cloudwatch_vpc_flow_log_role`?

Reply:
Gone for cloudwatch_vpc_flow_log_assume_role